packages/instrumentation-aws-sdk/src/services/sqs.ts

```diff
@@ -75,8 +70,7 @@ export class SqsServiceExtension implements ServiceExtension {
           isIncoming = true;
           spanKind = SpanKind.CONSUMER;
           spanName = `${queueName} receive`;
-          spanAttributes[SEMATTRS_MESSAGING_OPERATION] =
-            MESSAGINGOPERATIONVALUES_RECEIVE;
+          spanAttributes[ATTR_MESSAGING_OPERATION_TYPE] = 'receive';
```

Comment:
nit: There *are* enum value exports from the unstable semantic-conventions exports that could be used for 'receive' here (and `aws_sqs` above):

```
8674:export const MESSAGING_OPERATION_TYPE_VALUE_CREATE = "create" as const;
8685:export const MESSAGING_OPERATION_TYPE_VALUE_DELIVER = "deliver" as const;
8694:export const MESSAGING_OPERATION_TYPE_VALUE_PROCESS = "process" as const;
8705:export const MESSAGING_OPERATION_TYPE_VALUE_PUBLISH = "publish" as const;
8714:export const MESSAGING_OPERATION_TYPE_VALUE_RECEIVE = "receive" as const;
8723:export const MESSAGING_OPERATION_TYPE_VALUE_SEND = "send" as const;
8732:export const MESSAGING_OPERATION_TYPE_VALUE_SETTLE = "settle" as const;
```

However, this is very minor and could be done in a follow-up PR.